docs/reference/config.toml

```diff
@@ -1,4 +1,4 @@
-baseURL = "/mongo-java-driver"
+baseURL = "/mongo-java-driver/4.0"
```

Comment:
What's the effect of this change?

Reply:
This reverts back the change that made the 4.0 documentation sit on the top URL "/mongo-java-driver" allowing the landing page to be reinstated.

Reply:
I think it's doing something else too, though not sure: the "MONGODB JVM DRIVERS" link at the top left of the 4.0 reference docs links to http://rosslawley.co.uk/mongo-java-driver/4.0/ instead of back to the landing page, as it does on the current docs.

Reply:
https://github.com/rozza/mongo-java-driver/blob/master/docs/reference/config.toml#L1

Reply:
Found it: https://github.com/mongodb/mongo-java-driver/blob/master/docs/reference/themes/mongodb/layouts/partials/menu.html#L6

Reply:
Nice